addons/binding/org.openhab.binding.roku/OSGI-INF/RokuDiscovery.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2017 by the respective copyright holders.
```

Comment:
Some times has passed by can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.